pillar.example

```diff
@@ -260,7 +260,7 @@ openvpn:
       # yamllint disable-line rule:line-length
       tls_cipher: 'DHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-SHA256:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-SHA256:DHE-RSA-CAMELLIA256-SHA:DHE-RSA-AES256-SHA:DHE-RSA-CAMELLIA128-SHA:DHE-RSA-AES128-SHA:CAMELLIA256-SHA:AES256-SHA:CAMELLIA128-SHA:AES128-SHA'
       remote_cert_tls: server
-      comp_lzo:
+      comp_lzo: "lzo"
```

Comment:
https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/

> `--comp-lzo [mode]`
> **DEPRECATED** This option will be removed in a future OpenVPN release. Use the newer `--compress` instead.Use LZO compression — may add up to 1 byte per packet for incompressible data. `mode` may be `"yes"`, `"no"`, or `"adaptive"` (default).

* `"lzo"` is not one of the available options for `comp-lzo` but it is an option for its replacement, `compress`.

Reply:
Sorry about that... it's fixed now!